src/runtime/auto-prompt-manager.js

```diff
@@ -258,7 +258,7 @@ export class AutoPromptManager {
     }
 
     // Fetched config returned no maximum cap.
-    if (autoPromptConfig.maxImpressionsPerWeek === undefined) {
+    if (autoPromptConfig.impressionConfig.maxImpressions === undefined) {
```

Comment:
optional chaining?
```
if (autoPromptConfig.impressionConfig?.maxImpressions === undefined) {
```

Reply:
Probably not needed here since autoPromptConfig.impressionConfig is never undefined.